Makefile

```diff
@@ -93,6 +93,7 @@ CONTROLLER_GEN = $(LOCAL_BIN)/controller-gen
 KUSTOMIZE = $(LOCAL_BIN)/kustomize
 ENVTEST = $(LOCAL_BIN)/setup-envtest
 GO_BINDATA = $(LOCAL_BIN)/go-bindata
+# This must be in sync with the go.mod file.
 KUSTOMIZE_VERSION ?= v5.0.1
```

Comment:
Can we parse it from the go.mod file, then? Something like this like we do with Ginkgo?
```suggestion
# This must be in sync with the go.mod file.
KUSTOMIZE_VERSION ?= $(shell awk '/sigs.k8s.io\/kustomize\/kustomize\/v/ {print $$2}' go.mod)
```